### ISSUE TITLE
Partition visualizer not accessible to keyboard users

### ISSUE THREAD

**Issue by orta** [elementary/installer#549]
**Describe the bug**

Hey folks, without access to a mouse it's not possible to select a partition using the graph. This is tricky if you have a bluetooth mouse and haven't set it up yet during the install process on a real machine. 

This graph:
![Screen Shot 2021-05-16 at 11 50 47 AM](https://user-images.githubusercontent.com/49038/118394489-f6369f80-b63c-11eb-8225-307324dec61c.png)

The workaround is to go to "Try Demo Mode", use search to get to the bluetooth panel, add your mouse, then open "Install Elementary"

**To Reproduce**
1. Install beta, go through keyboard options
2. Click on Custom (Advanced)
3. Try to use the keyboard to select your partition

**Expected behavior**
Active states, and pressing enter/space to trigger the popover

**Desktop (please complete the following information):**
 - OS: e6 beta
 - Browser N/A
